superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -1542,12 +1546,14 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
             >
               <div className="control-label">{t('Screenshot width')}</div>
               <div className="input-container">
-                <Input
+                <InputNumber
                   type="number"
                   name="custom_width"
-                  value={currentAlert?.custom_width || ''}
+                  value={currentAlert?.custom_width ?? null}
```

Comment:
Given that we're setting 600 as the minimum value, should accept `null` values here or always start with 600?

Reply:
I think it makes sense to have 600 as a default value here. Perhaps we could export in a const for better visibility

Reply:
On the backend, custom_width as a field is nullable, and when null, it defaults to the WEBDRIVER_WINDOW in config.py

Default value is:
```
 WEBDRIVER_WINDOW = {
    "dashboard": (1600, 2000),
    "slice": (3000, 1200),
    "pixel_density": 1,
} 
```

Reply:
@fisjac for the ts error here, I think you're going to want the default to be `undefined`
`error TS2322: Type 'number | null' is not assignable to type 'number | undefined'.`

Reply:
This will never be below 600 though right and it won't ever be null or undefined for that reason too?